src/common/imagpng.cpp

```diff
@@ -23,6 +23,10 @@
 
 #if wxUSE_IMAGE && wxUSE_LIBPNG
 
+#include "../png/png.h"
```

Comment:
Aren't these headers private to libpng? I think only `png.h` is supposed to be included by the code using the library, why do we need to include those?